contents/blog/posthog-vs-amplitude.mdx

```diff
@@ -1,145 +1,241 @@
 featuredImageType: full
-author:
-  - andy-vandervell
-  - joe-martin
 category: General
 tags:
   - Comparisons
 ---
 
 import { ComparisonTable } from 'components/ComparisonTable'
+import { ComparisonHeader } from 'components/ComparisonTable/header'
 import { ComparisonRow } from 'components/ComparisonTable/row'
 
-**Contents:**
+Want to know how PostHog and Amplitude compare? If you remember nothing else, remember these two things:
 
-- [How is PostHog different to Amplitude?](#how-is-posthog-different-from-amplitude)
-- [PostHog vs Amplitude feature comparison](#feature-comparison)
-- [Integrations and data sources](#integrations-and-data-sources)
-- [Strengths of PostHog](#strengths-of-posthog)
-- [Strengths of Amplitude](#strengths-of-amplitude)
+1. **Amplitude** is a product analytics tool built for non-technical users and data analysts.
 
-## How is PostHog different from Amplitude?
+2. **PostHog** is an all-in-one platform built for engineers that replaces Amplitude _and_ numerous other tools.
 
-Over 10,000 companies already use PostHog, and many have switched from Amplitude. Here are a few reasons why.
+## How is PostHog different?
 
-### 1. It's an all-in-one platform
+### 1. We're everything you need in one
 
-PostHog is more than a product analytics tool, it's an all-in-one Product OS that replaces multiple tools in your data stack. You could run Amplitude for analytics, [LaunchDarkly](/blog/posthog-vs-launchdarkly) for feature flagging, Hotjar for session recording, and [Optimizely](/blog/posthog-vs-optimizely) for A/B testing, or you could just use PostHog for everything. One platform, one price: seamless integration.
+PostHog replaces multiple tools in your data stack. You could run Amplitude for analytics, [LaunchDarkly](/blog/posthog-vs-launchdarkly) for feature flags and testing, [Hotjar](/blog/posthog-vs-hotjar) for session replays and surveys, or you could just use PostHog for everything. One platform, one bill, and seamless integration.
 
-### 2. It's built for engineers
+### 2. We're built for engineers
 
-Unlike Amplitude, PostHog is built for software developers. PostHog autocaptures data, so you don't have to spend time instrumenting events every time you update your app or website – we also offer robust tools for capturing custom events and actions when you need to. Our pricing is transparent, and we're entirely self-serve. You can setup PostHog and start paying us without ever speaking to a sales rep, but we have an [awesome customer success team](/contact-sales) if you have questions.
+PostHog autocaptures data, so you don't need to waste energy instrumenting basic events. You can choose between light and dark mode themes. Features like our custom SQL query builder, keyboard shortcuts, and command palette, cater to power users. You can setup PostHog and start paying us without ever speaking to a sales rep, though our [customer success team](/contact-sales) is always available to answer your questions.
```

Comment:
```suggestion
We support all popular (and some unpopular) SDKs, have extensively documented APIs, and a powerful SQL query builder, so you can analyze data how you want. Because we're all-in-one, our JavaScript snippet is all you need to implement multiple integrated tools. You can choose between US and EU data hosting. We don't publish white papers because they're lame.
```

Thoughts on this instead @ivanagas @Lior539?

Reply:
I like the first half, but not the second. Feels like it takes you in a different direction. Here's my suggestion (I've also added more hyperlinks)

```suggestion
We support all popular (and some unpopular) [SDKs](/docs/libraries), have [extensively documented APIs](/docs/api), and a [powerful SQL query builder](/docs/hogql), so you can analyze data how you want. 

We also offer tools that enable you to [safely test in production](/product-engineers/testing-in-production), [phased rollouts](/tutorials/phased-rollout), links to other tutorials
```

Reply:
A bit too long, but I'll work with it.